docs/source/docker/getting_started/hardware_recommendations.rst

```diff
@@ -0,0 +1,82 @@
+.. _ref-hardware-recommendations:
+
+Hardware recommendations
+========================
+
+Usually, the Lightly worker is run on a cloud machine,
```

Comment:
Usually, the Lightly worker is run on a cloud machine, ->
Lightly worker is usually run on dedicated hardware or in the cloud on a compute instance which is specifically spun up to run Lightly Worker standalone.